rpm_spec/qubes-desktop-linux-manager.spec.in

```diff
@@ -61,6 +61,8 @@ Requires:  qubes-artwork >= 4.1.5
 Requires:  xhost
 Requires:  gtksourceview4
 Requires:  gnome-icon-theme
+Requires:  xfce4-settings
```

Comment:
This package shouldn't be necessary, after all you can use it under KDE. What breaks without it? The linked issue says it's icons, but at least in R4.3 we have new custom icons created for domains widget specifically, so they couldn't possibly be provided by xfce.